decentraland/unity-renderer
# feat: leave channel

## What does this PR change?

Add the ability to leave channels

## How to test the changes?

1. Open the conversation list
2. Click the 3 dots button over the entry
3. Click "Leave" option
4. The channel should dissapear from the list
5. Open any channel
6. Click the 3 dots button over the header of the window
7. Click "Leave" option
8. The window should be closed, the conversation should be opened having removed the channel
9. Open any channel
10. Type `/leave`
11. The window should be closed, the conversation should be opened having removed the channel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
